libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/camera/DynamicCamera.java

```diff
@@ -22,8 +22,8 @@
 
   private static final double MAX_CAMERA_TILT = 60d;
   private static final double MIN_CAMERA_TILT = 45d;
-  private static final double MAX_CAMERA_ZOOM = 16d;
-  private static final double MIN_CAMERA_ZOOM = 12d;
+  private static final double MAX_CAMERA_ZOOM = 20d;
```

Comment:
Have we tested these new values and the experience is better? Why `20d` instead of `16d`? Same for `MIN_CAMERA_ZOOM` cc @LukasPaczos

Reply:
as @kmadsen mentioned https://github.com/mapbox/mapbox-navigation-android/pull/2437#discussion_r372433266 based on users' experience. I made it that 18 will be middle of 16(min) and 20(max)

Reply:
I'm asking because the `DynamicCamera` has its own logic to change the zoom based on the `RouteProgress` and changing these values affect the overall experience.

Reply:
actually I didn't note any changes new vs old approach(run with test-app and route-simulator)

Reply:
revert to 16 and 12 cc @LukasPaczos @Guardiola31337 @kmadsen